framework/src/ble_manager/ble_manager_state.c

```diff
@@ -913,6 +913,7 @@ ble_result_e blemgr_handle_request(blemgr_msg_s *msg)
 		if (ctx == NULL) {
 			BLE_LOG_ERROR("[BLEMGR] fail to find BLE context table");
 			ret = TRBLE_NOT_FOUND;
+			free(msg->param);
```

Comment:
From here, we don't use `msg->param` anymore.
How about free it befor this if statement?
Then we don't need to call free from line 932

Reply:
@jeongchanKim 
The value of 'msg->param'  is allocated in 'ble_device_connected *data'.
It keeps using until the end of 'BLE_EVT_CLIENT_CONNECT' case